proxy/src/context.rs

```diff
@@ -42,15 +43,16 @@ define_result!(Error);
 /// Context for request, may contains
 /// 1. Request context and options
 /// 2. Info from http headers
+#[derive(Debug)]
 pub struct RequestContext {
```

Comment:
What's the relationship between `RequestContext` and `Context`(in proxy/src/lib.rs)?
I found the fileds in them are similar, can we just keep one?

Reply:
RequestContext is mainly used in HTTP handler, and convert to Context to use.

Maybe we can combine them into one. I can try this in another PR.